src/vs/platform/theme/common/colorRegistry.ts

```diff
@@ -190,8 +190,23 @@ export const activeContrastBorder = registerColor('contrastActiveBorder', { ligh
 
 export const selectionBackground = registerColor('selection.background', { light: null, dark: null, hc: null }, nls.localize('selectionBackground', "The background color of text selections in the workbench (e.g. for input fields or text areas). Note that this does not apply to selections within the editor."));
 
-// ------ text colors
+// -- Welcome Page Colors
```

Comment:
please add a `// {{SQL CARBON EDIT}}` comment above any code changes in the `vs` directory?

@anthonydresser is this the best way to add these settings?  Is there a way to add these changes somewhere in the welcome page or under `sql`

Reply:
No reason these need to be here just make the same file path under the sql folder and add these there (`sql/platform/theme/common/colorRegistry.ts`)

Reply:
removed changes to vs folder, added to sql folder